src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -368,4 +370,16 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
     this.itemForm.enable();
     this.editContent?.reset();
   }
+
+  private maxTeamSizeValidator(): ValidatorFn {
+    return (itemForm): null => {
+      const isParticipationAsTeamOnly = itemForm.get('entry_participant_type')?.value as boolean;
+      const maxTeamSizeControl = itemForm.get('entry_max_team_size');
+      if (!maxTeamSizeControl) return null;
+
+      const errors = isParticipationAsTeamOnly ? Validators.min(1)(maxTeamSizeControl) : null;
+      maxTeamSizeControl.setErrors(errors);
```

Comment:
```suggestion
      if (isParticipationAsTeamOnly) maxTeamSizeControl.setErrors(Validators.min(1)(maxTeamSizeControl));
```
would have be more readable from my point of view

Reply:
this sample doesn't cover the case when switching from participation as team to the opposite, but I get the idea, I'll change it accordingly.

Reply:
Fixed